src/legacy/core_plugins/kibana/public/management/sections/settings/components/field/field.test.tsx

```diff
@@ -263,6 +276,7 @@ describe('Field', () => {
           />
         );
         const select = findTestSubject(component, `advancedSetting-editField-${setting.name}`);
+        // @ts-ignore
```

Comment:
nit: Can we remove this and use casting, if needed?

Reply:
I'm happy to do this if you can show me which type is needed as I've been unable to find a good example.

Reply:
```ts
const labels = select.find('option').map((option: any) => option.text());
```

or 

```ts
const labels = select.find('option').map((option: { text: () => string[] }) => option.text());
```

Reply:
I'm a little confused about this. Is this useful aside from removing `// @ts-ignore`?

Reply:
No, my comment was just about removing `// @ts-ignore`. It is just a nit.